src/labels/CustomLabels.labels

```diff
@@ -7253,7 +7439,7 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Warning about some fields when donation already has a GAU Allocation assigned</shortDescription>
-        <value>Since this donation has at least one GAU Allocation assigned, you can&apos;t update certain fields.</value>
+        <value>You can&apos;t update certain fields because this donation has at least one GAU Allocation assigned.</value>
```

Comment:
```suggestion
        <value>You can't update certain fields because this donation has at least one GAU Allocation assigned.</value>
```

Reply:
@ptewson-sfdo I'm not sure this change is correct, but I've seen some instances of the code for an apostrophe and others with just a plain text apostrophe. I'm good with whichever is best practice.